packages/gatsby/src/utils/start-server.ts

```diff
@@ -92,20 +92,12 @@ export async function startServer(
     !isCI()
   ) {
     cancelDevJSNotice = showExperimentNoticeAfterTimeout(
-      `PRESERVE_WEBPACK_CACHE`,
-      report.stripIndent(`
-Your friendly Gatsby maintainers detected your site has more JavaScript than most sites! We're working to make your site's JS compile as quickly as possible by avoiding clearing your webpack cache as often.
+      `Preserve webpack's Cache`,
+      `https://github.com/gatsbyjs/gatsby/discussions/28331`,
+      `which changes Gatsby's cache clearing behavior to not clear webpack's cache unless really
+necessary. Here's how to try it:
```

Comment:
Is this true? We currently never clear webpack cache with this flag - only user can do manual `gatsby clean` or `rm -rf .cache` that would clear it (I think?)

Reply:
yeah, that's more explicit (so more helpful) — I'll change it :+